src/engraving/layout/layoutsystem.cpp

```diff
@@ -1187,10 +1191,14 @@ void LayoutSystem::layoutSystemElements(const LayoutOptions& options, LayoutCont
 void LayoutSystem::processLines(System* system, std::vector<Spanner*> lines, bool align)
 {
     std::vector<SpannerSegment*> segments;
+    std::vector<SlurSegment*> slurSegments;
```

Comment:
instead of filling and allocating a new vector of these pointers, you could just put a check to the body of the 'for loop" at 1234